test/e2e/move.new.proposed.test.js

```diff
@@ -0,0 +1,59 @@
+import { Selector } from 'testcafe'
+
+import { newMove } from './_routes'
+import { OCAUser } from './roles'
+import { page, moveDetailPage, createMovePage } from './pages'
+import { createPersonFixture } from './helpers'
+
+fixture('New proposed move').beforeEach(async t => {
+  await t.useRole(OCAUser).navigateTo(newMove)
+})
+
+test('Prison to prison as proposed move', async t => {
```

Comment:
I think it might be nicer to have these within the existing `move.new.prison.test.js` file to keep things consistent with how we do the Police journeys.

The files are split based on from location, and then the fixtures/tests within those use the different journeys.